examples/04_vcb_tool/README.md

```diff
@@ -0,0 +1,6 @@
+# Measurement Bar Tool
```

Comment:
I like the separation - makes it so much clearer to what the example is focusing on.